src/components/Person/Person.jsx

```diff
@@ -1 +1,27 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+function Person({ person }) {
```

Comment:
Just write export here
```suggestion
export function Person({ person }) {
```